ntoskrnl/mm/ARM3/miarm.h

```diff
@@ -1029,6 +1029,10 @@ MI_WS_OWNER(IN PEPROCESS Process)
 //
 // New ARM3<->RosMM PAGE Architecture
 //
+#ifdef __clang__ /* FIXME: Inspect */
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmultichar"
+#endif
```

Comment:
And this one as well.

Reply:
Actually, we disable both globally in gcc builds so I think I'll remove these local ones and opt for the global ones.